Source/WebKit/UIProcess/WebFrameProxy.h

```diff
@@ -191,6 +204,8 @@ class WebFrameProxy : public API::ObjectImpl<API::Object::Type::Frame>, public C
     WebCore::FrameIdentifier m_frameID;
     ListHashSet<Ref<WebFrameProxy>> m_childFrames;
     WeakPtr<WebFrameProxy> m_parentFrame;
+    WeakPtr<WebFrameProxy> m_nextSibling;
+    WeakPtr<WebFrameProxy> m_previousSibling;
```

Comment:
I feel like these are redundant with having a ListHashSet of children.  To find the next sibling, you go to the parent's children, find(*this) gives you an iterator, and you can check if it's the end and increment if not.  I guess ListHashSet needs an rfind that gives you a reverse_iterator.

Reply:
FrameTree in WebCore stores sibling frames as member variables, so I was just following the pattern. But I guess we don't need to, I'll change it.